### PR TITLE
Correct module name in htaccess

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30649.

### Summary of Changes

Corrects module name.

### Testing Instructions

Code review.

Or test on Apache with module `mod_autoindex.c` enabled:

Rename `htaccess.txt` to `.htaccess`.
Create an empty directory or remove `index.html` from existing directory.
Open directory in browser.

### Actual result BEFORE applying this Pull Request

Directory contents listed.

### Expected result AFTER applying this Pull Request

Directory contents not listed.

### Documentation Changes Required

No.

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 8e0a67b5488fcbbf3019923eb2493afaa33af3c0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30655">issues.joomla.org/tracker/joomla-cms/30655</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 8e0a67b5488fcbbf3019923eb2493afaa33af3c0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30655">issues.joomla.org/tracker/joomla-cms/30655</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30655">issues.joomla.org/tracker/joomla-cms/30655</a>.</sub>

**Comment by sandewt** [comment]
Question: will this change also be mentioned in the next update of Joomla in the Post-installation Messages for Joomla CMS?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30655">issues.joomla.org/tracker/joomla-cms/30655</a>.</sub>

**Comment by richard67** [comment]
> Question: will this change also be mentioned in the next update of Joomla in the Post-installation Messages for Joomla CMS?

No idea. Ping @HLeithner @zero-24 .

**Comment by HLeithner** [comment]
Only if @SharkyKZ creates a post installation message, alternative we can write it into the release article.

@SniperSister any objections on this.

**Comment by sandewt** [comment]
>Only if @SharkyKZ creates a post installation message, alternative we can write it into the release article.

As a reminder, joomla 3.9.21 also included a post-install message to change the .htaccess file.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30655">issues.joomla.org/tracker/joomla-cms/30655</a>.</sub>

**Comment by SniperSister** [comment]
Fine for me!

**Comment by SharkyKZ** [comment]
I'll look into including a postinstall message. Removing RTC now.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30655">issues.joomla.org/tracker/joomla-cms/30655</a>.</sub>

**Comment by SharkyKZ** [comment]
Postinstall message added.

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on 941fce5db773a58ae5ca42eb6d0debde3d72eb5a<br /><br />Joomla! 3.9.22-dev+pr.30655 Development [ Amani ] 26-August-2020 08:12 GMT
mysql (5.5.5-10.4.14-MariaDB)

Tested: 
`mod_autoindex.c`
and
`post install message` 

Questing: why this addition _as this file can not be updated automatically_ ?

[EDIT]
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30655">issues.joomla.org/tracker/joomla-cms/30655</a>.</sub>

**Comment by SharkyKZ** [comment]
>Questing: why this addition as this file can not be updated automatically ?

I just copied this from the other message. We don't modify existing `.htaccess` file, users have to do it themselves.

**Comment by sandewt** [comment]
>As a reminder, joomla 3.9.21 also included a post-install message to change the .htaccess file.

This message has now disappeared in Joomla 3.9.22 is that the intention?

**Comment by SharkyKZ** [comment]
The 3.9.21 message is added only when updating to 3.9.21. It is not shown on new installations or when running on git clone or updating from nightly to nightly. You'd have to manually to run the SQL script to have it added to the database.

**Comment by sandewt** [comment]
@SharkyKZ thank you for the explanation.

**Comment by richard67** [comment]
@SharkyKZ Postinstall message is not described in summary of changes and testing incstructions.

File: administrator/language/en-GB/en-GB.com_admin.ini [hunk 1]
```diff
@@ -136,6 +136,8 @@ COM_ADMIN_PHP_SETTINGS="PHP Settings"
 COM_ADMIN_PHP_VERSION="PHP Version"
 COM_ADMIN_PHPINFO_DISABLED="The built in phpinfo() function has been disabled by your host."
 COM_ADMIN_PLATFORM_VERSION="Joomla! Platform Version"
+COM_ADMIN_POSTINSTALL_MSG_HTACCESS_AUTOINDEX_DESCRIPTION="<p>Before 3.9.22 default htaccess.txt file contained erroneous code meant for disabling directory listings. The security team recommends to manually apply the necessary changes to any existing .htaccess file, as this file can not be updated automatically.</p><p>The old code:</p><pre>&lt;IfModule autoindex&gt;\n  IndexIgnore *\n&lt;/IfModule&gt;</pre><p>The new code:</p><pre>&lt;IfModule mod_autoindex.c&gt;\n  IndexIgnore *\n&lt;/IfModule&gt;</pre>"
```
Comment (richard67) [inline administrator/language/en-GB/en-GB.com_admin.ini]:
```suggestion
COM_ADMIN_POSTINSTALL_MSG_HTACCESS_AUTOINDEX_DESCRIPTION="<p>Before 3.9.22 the default htaccess.txt file contained erroneous code meant for disabling directory listings. The security team recommends to manually apply the necessary changes to any existing .htaccess file, as this file can not be updated automatically.</p><p>The old code:</p><pre>&lt;IfModule autoindex&gt;\n  IndexIgnore *\n&lt;/IfModule&gt;</pre><p>The new code:</p><pre>&lt;IfModule mod_autoindex.c&gt;\n  IndexIgnore *\n&lt;/IfModule&gt;</pre>"
```

Comment (richard67) [reply administrator/language/en-GB/en-GB.com_admin.ini]: Change "Before 3.9.22 default htaccess.txt file ..." to "Before 3.9.22 **the** default htaccess.txt file ...".

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 8176b377ef71033a087faa46ad324d37303b0d36<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30655">issues.joomla.org/tracker/joomla-cms/30655</a>.</sub>

**Comment by richard67** [comment]
Previous test by @sandewt was still valid because change after that was only a small improvement of the language string. I've restored the test in the issue tracker so it is counted correctly in the tracker and here.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30655">issues.joomla.org/tracker/joomla-cms/30655</a>.</sub>

**Comment by HLeithner** [comment]
Thanks

**Pull request merged**